esbuild/plugins.ts

```diff
@@ -146,7 +146,6 @@ function processManifestPlugin({
             scripts: [json.background.service_worker]
           }
           json.content_scripts?.forEach((contentScript) => {
-            // @ts-expect-error firefox doesn't support execution context yet
```

Comment:
Let's add a TODO comment here, that we should remove this once WORLD is supported in Firefox like last 10 versions.